packages/components/src/Enumeration/Items/Items.component.js

```diff
@@ -51,9 +55,13 @@ class Items extends React.PureComponent {
 				);
 			}
 			default: {
+				let actions = this.props.itemsProp.actionsDefault;
+				if (typeof actions === 'function') {
+					actions = actions(itemWithIndex);
+				}
 				const itemPropDefault = {
 					key: this.props.itemsProp.key,
-					actions: this.props.itemsProp.actionsDefault,
+					actions,
 					onSelectItem: this.props.itemsProp.onSelectItem,
```

Comment:
looks like changes in both cases are completely the same and can be done once above switch

Reply:
The difference is when in default mode, it uses `actionsDefault`; in edit mode, it uses `actionsEdit`.

Reply:
ah, yes, good